frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1869,20 +1887,38 @@ function ReaderFooter:_updateFooterText(force_repaint, force_recompute)
     end
 end
 
+function ReaderFooter:onChangeViewMode()
+    self:onPageUpdate()
```

Comment:
May be a comment about why you need to call it.
(I have the feeling it would be called anyway because of other events, but better explicite than implicite if it being called multiple times is not an issue.)